client/network/bitswap/build.rs

```diff
@@ -1,4 +1,4 @@
-const PROTOS: &[&str] = &["bitswap.v1.2.0.proto"];
+const PROTOS: &[&str] = &["src/schema/bitswap.v1.2.0.proto"];
```

Comment:
I'm fine with this change because it seems to work either way and for some reason the [example code](https://docs.rs/prost-build/latest/prost_build/index.html) `prost-build` also adds the include path to the source files. But it would be interesting to know why the code as it was before is not working for you.

Reply:
If we look some lines below, we give the path `src/schema` as includes. Looking at the docs, the files given in `PROTOS` should be found in the given includes. So, it should work. 

So, I also want to know why it doesn't work for you and then we need to report this to upstream.

Reply:
https://docs.rs/prost-build/latest/prost_build/fn.compile_protos.html 

If you read the docs there, we do it exactly as explained in the docs. Others have reported that maybe deleting the `target` dir fixed it for them. There is clearly some bug, but I don't think in our code. None of our devs also reported this issue.

Reply:
Yes, I read this document. I will ask the developer of prost-build for the usage of this parameter.

Reply:
I asked PROST about the issue, and maybe they can answer the questions.    https://github.com/tokio-rs/prost/issues/734

Reply:
Ty @anonymousGiga :heart:

Reply:
I got a response from this issue（https://github.com/tokio-rs/prost/issues/734） ,  may be we should use "```src/schema/bitswap.v1.2.0.proto```" instead of "```bitswap.v1.2.0.proto```".   @bkchr

Reply:
@anonymousGiga could you then please update all usages of `compile_protos`?

Reply:
I checked that all the other places where  ```compile_protos```  is used are correct, and they all use in the way "prefix the protos with the path".     In addition, I also updated to the latest master branch.    @bkchr